godtools/App/Services/Renderer/Views/Tool/Views/Header/ToolPageHeaderView.swift

```diff
@@ -45,8 +45,10 @@ class ToolPageHeaderView: MobileContentView {
         
         backgroundColor = .clear
         backgroundView.backgroundColor = viewModel.backgroundColor
+        
+        backgroundView.semanticContentAttribute = viewModel.languageDirectionSemanticContentAttribute
```

Comment:
That fixed it!  For some reason it doesn't seem to work anymore when setting directly on the ToolPageHeaderView.  I'm not sure why that is.  Better to just apply to both of those views.

Reply:
Yeah it must be treating the backgroundView as the parent of the title and number, not the ToolPageHeaderView.